### PR TITLE
Mobileview modification and error handling 

### PR DESCRIPTION
**Pull request by srikanthbalagam**
loan propser details page error handling is completed .......

and also created a   isTitleholder same as loanproposer page is created

### CODE REVIEW

File: public/frontendimg2.jpg
Comment (Praveen238) [inline public/frontendimg2.jpg:1]: ok, this background image is good for the home page

File: src/pagecomponents/LoanProposerDetails.js
Comment (Praveen238) [inline src/pagecomponents/LoanProposerDetails.js:1]: i think some of the fields are not required for every person

**Review by Praveen238 — APPROVED** [review]
ok overall good

File: src/pagecomponents/LoanProposerDetails.js
Comment (srikanthbalagam) [reply src/pagecomponents/LoanProposerDetails.js:1]: i will try to make changes on that

**Pull request merged**